acceptance-tests/acceptance-tests.ts

```diff
@@ -17,242 +17,251 @@ const TestCaseId = {
-
-      browser.assert.text('h1', 'Authentication failed!')
-      browser.assert.text('p', /Because you cancelled/)
-    })
-  })
-
-  describe('On receiving an authn failed response', () => {
-    let testCaseUri: string
-
-    beforeEach(() => {
-      setupComplianceTool(`pid`)
-    })
-
-    beforeEach(() => {
-      return makeAuthnRequest()
-        .then(() => getTestCaseUri(TestCaseId.AUTHENTICATION_FAILED))
-        .then(uri => testCaseUri = uri)
-    })
-
-    it('handles the response correctly', async () => {
-      await browser.visit(testCaseUri)
-
-      browser.assert.text('h1', 'Authentication failed!')
-      browser.assert.text('p', /Because no user matching your credentials could be found/)
-    })
-  })
-
-  describe('On receiving an error response', () => {
-    let testCaseUri: string
-
-    beforeEach(() => {
-      setupComplianceTool(`pid`)
-    })
-
-    beforeEach(() => {
-      return makeAuthnRequest()
-        .then(() => getTestCaseUri(TestCaseId.REQUESTER_ERROR))
-        .then(uri => testCaseUri = uri)
-    })
-
-    it('handles the response correctly', async () => {
-      await browser.visit(testCaseUri)
-
-      browser.assert.text('h1', 'Something went wrong')
-      browser.assert.text('p', /There was an error while trying to verify your identity\./)
-    })
-  })
-
-  describe('On receiving a user account creation response', () => {
-    let testCaseUri: string
-
-    beforeEach(() => {
-      setupComplianceTool(`unseen-pid-${Date.now()}`)
-    })
-
-    beforeEach(() => {
-      return makeAuthnRequest()
-        .then(() => getTestCaseUri(TestCaseId.ACCOUNT_CREATION_LOA2))
-        .then(uri => testCaseUri = uri)
-    })
-
-    it('handles the response correctly', async () => {
-      await browser.visit(testCaseUri)
-
-      browser.assert.text('h1', 'Success!')
-
-      browser.assert.text('p', /Screaming/)
-    })
-  })
-
-  describe('On receiving an incorrectly signed response', () => {
-    let testCaseUri: string
-
-    before(() => {
-      setupComplianceTool('pid')
-    })
-
-    beforeEach(() => {
-      return makeAuthnRequest()
-        .then(() => getTestCaseUri(TestCaseId.BASIC_SUCCESSFUL_MATCH_WITH_ASSERTIONS_SIGNED_BY_HUB))
-        .then(uri => testCaseUri = uri)
-    })
-
-    it('serves an appropriate error', async () => {
-      await browser.visit(testCaseUri)
-
-      browser.assert.text('h1', 'Something went wrong')
-      browser.assert.text('p', /There was an error while trying to verify your identity\./)
+let VERIFY_SERVICE_PROVIDER_HOST = process.env['VERIFY_SERVICE_PROVIDER_HOST']
+if (!VERIFY_SERVICE_PROVIDER_HOST) {
+  console.log('Warning: VERIFY_SERVICE_PROVIDER_HOST not set, using localhost:50400 by default. Use the --paas flag to run against the service provider on paas.')
+  VERIFY_SERVICE_PROVIDER_HOST = 'http://localhost:50400'
+}
+const COMPLIANCE_TOOL_HOST = 'https://compliance-tool-reference.ida.digital.cabinet-office.gov.uk'
+
```

Comment:
N.B. The below are exactly the same tests as before, just moved inside the forEach which runs them for not providing an entityId and for providing one [which happens to be the sole entityId configured in the VSP set up on paas]. This isn't ideal, since it doesn't really by itself show that you can use different entity ids, because they both end up using the same one in the VSP, but this works with a single VSP running to test against.